epoxy-paging/src/main/java/com/airbnb/epoxy/paging/PagedListModelCache.kt

```diff
@@ -97,6 +97,11 @@ internal class PagedListModelCache<T>(
             (0 until count).forEach {
                 modelCache.add(position, null)
             }
+            lastPosition?.let {
+                if (position < it) {
```

Comment:
Why does it matter if `it + count` is greater than position? 
If `lastPosition` was 5 and we just inserted 50 items before 5th position e.g. `onInserted(position = 0, count = 50)` then your "lastPosition" MUST be adjusted to 50 + 5.

Reply:
`lastPosition` is the last index that was bound, not the last index that was added. They are too different things, and using the same property to represent two different things is confusing and dangerous.

Reply:
`lastPosition` can be renamed to `nextLoadAroundPosition`. When we insert new data in the list before this position, this variable no longer holds true. so renaming it may help. This way `nextLoadAroundPosition` acts like a command argument that should be used in next call to `loadAround`. This is exactly how it is being used now btw.